src/app/components/modal/index.html

```diff
@@ -179,6 +168,25 @@
 				<li>The form supports a complex task for a list of objects such as filling out report cards for a class.</li>
 				<li>The task includes subtasks that need to be completed in context.</li>
 			</ul>
+    
+		<div class="sky-section-heading">Content</div>
+		<div class="sky-subsection-heading">Modal title text</div>
+			<p>In general, modal titles should use sentence-case capitalization and the following format:</p>
+			<p><span class="sky-emphasized">&lt;Verb&gt; &lt;direct object&gt; for &lt;indirect object&gt;</span></p>
+			<div style="margin-bottom: 20px;">
+				<img class="screenshot screenshot-info" src="~/assets/img/guidelines/wording/modal-with-object.png" alt="modal title: Edit address for Robert Hernandez"/>
+			</div>
+			<p>If there is not an indirect object or the user is the implied indirect object, you can exclude it:</p>
+			<p><span class="sky-emphasized">&lt;Verb&gt; &lt;direct object&gt;</span></p>
+			<div style="margin-bottom: 20px;">
+				<img class="screenshot screenshot-info" src="~/assets/img/guidelines/wording/modal-without-object.png" alt="modal title: Manage bank feeds"/>
+			</div>
+		
+		<div class="sky-subsection-heading">Buttons in the modal action bar</div>
+			<ul>
+				<li>Use a primary <a routerLink="/components/button">button</a> for the most likely action the user will take. Use secondary buttons for less likely actions. Use a link button for the Cancel action.</li>
+				<li>If a modal has only one action, you should generally use the <a routerLink="/components/confirm">Confirm</a> component instead.</li>
```

Comment:
"confirm" should be lowercase. (Also, no big deal, but I'd include "the" and "component" as part of the link text so that it stands out a bit more. But that might just be me.) ... 

Also, can we delete "you should generally" to word this more strongly? Maybe there are exceptions to this rule, but if it's worth calling out, I think we should state it as a hard and fast rule rather than a recommendation. Again, just my 2 cents.

Reply:
There are exceptions for cases where there needs to be a lot of content on the modal.

Is something like "try using the Confirm component" still too wishy washy?

Reply:
I like that better, but if we want to avoid stating this as a hard and fast rule, I think either is fine. i made the (incorrect) assumption that the wording was an oversight and not an intentional thing. Sorry about that.

Reply:
I updated casing but left the verbiage as is. Let me know if that should've been changed after all.

Reply:
LGTM